pkg/issuer/acme/dns/acmedns/acmedns_test.go

```diff
@@ -47,7 +47,7 @@ func TestValidJsonAccount(t *testing.T) {
         "domain": {
             "fulldomain": "fooldom",
             "password": "secret",
-            "subdomain": "subdoom",
+            "subdomain": "subdom",
```

Comment:
it's possible someone was being cute, but most of this repository isn't particularly playful...